scio-extra/src/main/scala/com/spotify/scio/extra/hll/zetasketch/distributed/package.scala

```diff
@@ -0,0 +1,267 @@
+/*
+ * Copyright 2020 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.extra.hll.zetasketch
+
+import java.lang
+
+import com.google.protobuf.ByteString
+import com.google.zetasketch.HyperLogLogPlusPlus
+import com.spotify.scio.coders.Coder
+import com.spotify.scio.values.SCollection
+import com.twitter.algebird.{Monoid, MonoidAggregator}
+
+package object distributed {
+
+  // Distributed HyperLogLogPlusPlus impl support //
+  sealed trait HllPlus[T] extends Serializable {
```

Comment:
@syodage Sorry if I wasn't clear in my previous comment about the structure. It's ok for `HllPlus` to be under `com.spotify.scio.extra.hll.zetasketch` only but on a different file `HllPlus.scala` and all `syntax` extensions to `SCollection` under a `com.spotify.scio.extra.hll.zetasketch.syntax` package. Check for example the `csv` package under `scio-extra`.